cmd/common.go

```diff
@@ -408,6 +416,63 @@ func determineSelectorTag(targetContent file.Content, config dump.Config) ([]str
 	return config.SelectorTags, nil
 }
 
+func determineFilterChainSupport(ctx context.Context, client *kong.Client) (bool, error) {
+	workspace := client.Workspace()
+	client.SetWorkspace("")
+	defer client.SetWorkspace(workspace)
```

Comment:
This feels like a mega hack, but I get test failures without it because sometimes tests run against a workspace that doesn't exist. Is there a better way?

Reply:
I added a comment related to this here: https://github.com/Kong/deck/pull/987/files#r1629376245

In case we need to do this check, do you think we can modify the [fetchKongVersion](https://github.com/Kong/deck/blob/v1.38.1/cmd/common.go#L477) function to also deal with it?

Reply:
Ah, so `fetchKongVersion()` is actually where I got the idea for this. It does something similar by [trying to fetch / before trying the /{workspace} endpoint](https://github.com/Kong/deck/blob/526baa380db173161fe113a2580a2dca7f472155/cmd/common.go#L469-L485).

Reply:
I suggested to reuse `fetchKongVersion()` so that we will not have to run the same query twice.

Also, the current code would fail with 401 if decK doesn't have the permissions to query the default workspace.